src/blackboxTests/src/test/kotlin/org/vaccineimpact/api/blackboxTests/tests/BurdenEstimates/CloseBurdenEstimateSetTests.kt

```diff
@@ -109,8 +109,9 @@ class CloseBurdenEstimateSetTests : BurdenEstimateTests()
     {
         val requestHelper = RequestHelper()
 
-        val setId = JooqContext().use {
-            setUpWithBurdenEstimateSet(it)
+        val setId = 123
```

Comment:
This is another attempt at stopping that non-deterministic test failure, specifying the burden estimate set id to use so it doesn't just get assigned id `1`. Still can't really explain why it's happening unfortunately.

Reply:
Oh, so it gets assigned a different set id to the other test so they don't clash? Worth putting a comment in to that effect?